index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous" />
+    <title>Library</title>
+</head>
+
+<body>
+    <h1 class="text-center mt-5">A Library</h1>
+    <table id="table" class="table col-12 mt-5 mb-5">
+        <thead>
+            <th>
+                title
+            </th>
+            <th>
+                author
+            </th>
+            <th>
+                pages
+            </th>
+            <th>
+                read
+            </th>
+        </thead>
+        <tbody id="tbody"></tbody>
+
+    </table>
+    <div class="text-center">
+        <button class="btn btn-primary" id="form-btn">Add new book</button>
```

Comment:
The new book button should trigger an action to show the form, but currently, we have the button and the form showing even without pressing the button. This is not the expected behavior, let us kindly fix that

Reply:
@SamArsanios Please kindly address this.